rock_paper_scissors/rps.py

```diff
@@ -3,12 +3,22 @@
 import sys
 
 def rock_paper_scissors(n):
-  pass 
+  rps = ['rock', 'paper', 'scissors']
+  outcomes = []
 
+  def rps_results(roundsLeft, play):
+    if roundsLeft == 0:
+      return outcomes.append(play)
+    for i in rps:
+      rps_results(roundsLeft- 1, play + [i])
+  rps_results(n, [])
```

Comment:
Nice job on the recursive call here using the nested function. I did the same thing as well in the javascript code challenge.